<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLineEdit" name="lineEditPathTiLibrary">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>56</y>
     <width>433</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>20</y>
     <width>209</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Path to music folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFindPath">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>56</y>
     <width>53</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>252</y>
     <width>117</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>252</y>
     <width>117</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
